src/components/Input/DropDown/DropDownGroup.js

```diff
@@ -85,6 +86,14 @@ class DropDownGroup extends React.Component {
         this.toggleDropdown();
         break;
       default:
+        if (!isOpen) {
+          const { word } = this.state;
+          if (keyCode > 48 && keyCode < 90) {
```

Comment:
Can we move to seperate function to understand what exactly these codes mean?

Also, it's not clear for me, why `0` is not included (key code is 48) and small letters are not included?

Reply:
Good catch, 0 and z was not in range. 
We are checking keys, so there is no uppercase here, check https://keycode.info/